crypto/dilithium/pqcrystals_dilithium_ref_common/sign.c

```diff
@@ -300,7 +300,7 @@ int crypto_sign_signature(ml_dsa_params *params,
 }
 
 /*************************************************
-* Name:        crypto_sign
+* Name:        ml_dsa_sign_message
```

Comment:
`sign_message` and `verify_message` variants are never used (we use sign/verify without the appended message on the end, so that signature sizes are constant)-- should we just remove them from the library?